masterlist.yaml

```
@@ -18690,13 +18702,13 @@ plugins:
         condition: 'active("Better Dynamic Snow.esp") and not active("CWIBDSPatch.esp")'
       - <<: *patchIncluded
         subs: [ 'Cutting Room Floor' ]
-        condition: 'active("Cutting Room Floor.esp") and not active("CWICRFSEPatch.esp") and not active("UltimateCollege.esp")'
+        condition: 'active("Cutting Room Floor.esp") and not active("(CWICRFSEPatch|ICOW - (AIO - CRF Patch|CRF Patch))\.esp") and not active("UltimateCollege.esp")'
```

Since the patches are hosted on different pages & the regex doesn't simplify things much, I'd split them:

```suggestion
        condition: 'active("Cutting Room Floor.esp") and not active("CWICRFSEPatch.esp") and not active("ICOW - (AIO - )?CRF Patch\.esp") and not active("UltimateCollege.esp")'
```